.env.example

```
@@ -18,4 +18,7 @@ SPONSORSHIP_PICTURE_BUCKET=exampleKey
 WEBHOOK_SECRET=exampleKey
 NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=exampleKey
 NEXT_PUBLIC_SUPABASE_ANON_KEY=exampleKey
-NEXT_PUBLIC_SUPABASE_URL="https://bug-free-orbit-rq5vxrpqgv6357w9.github.dev/"
\ No newline at end of file
+NEXT_PUBLIC_SUPABASE_URL="https://bug-free-orbit-rq5vxrpqgv6357w9.github.dev/"
+NEXT_PUBLIC_TELEMETRY_SITE_URL=telemetrySiteUrl
+NEXT_PUBLIC_GRAFANA_URL=grafanaSiteUrl
+ 
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)